client/my-sites/themes/theme-showcase.jsx

```diff
@@ -119,7 +119,7 @@ const ThemeShowcase = React.createClass( {
 	},
 
 	render() {
-		const { site, options, getScreenshotOption, search, filter, translate } = this.props;
+		const { site, options, getScreenshotOption, search, filter, translate, siteSlug, isMultisite } = this.props;
```

Comment:
If we continue to add more vars here, we should probably start to put them on separate lines. 😄 Fine for now tho.